charts.py

```diff
@@ -0,0 +1,230 @@
+            main_chart_data[f"liquidable_debt_at_interval_{protocol}"] = (
+                protocol_main_chart_data["liquidable_debt_at_interval"]
+            )
+        else:
+            main_chart_data["liquidable_debt"] += protocol_main_chart_data[
+                "liquidable_debt"
+            ]
+            main_chart_data["liquidable_debt_at_interval"] += protocol_main_chart_data[
+                "liquidable_debt_at_interval"
+            ]
+            main_chart_data[f"liquidable_debt_{protocol}"] = protocol_main_chart_data[
+                "liquidable_debt"
+            ]
+            main_chart_data[f"liquidable_debt_at_interval_{protocol}"] = (
+                protocol_main_chart_data["liquidable_debt_at_interval"]
+            )
+
+        if loans_data.empty:
+            loans_data = protocol_loans_data
+        else:
+            loans_data = pandas.concat([loans_data, protocol_loans_data])
+
+        return main_chart_data, loans_data
+
+
+def get_protocol_main_data(
+    zklend, nostra_alpha, nostra_mainnet, current_pair, stable_coin_pair
+):
+    protocol_main_chart_data_mapping = (
+        {
+            ZKLEND: app.create_stablecoin_bundle(zklend)[current_pair],
+            # 'Hashstack V0': hashstack_v0_main_chart_data[current_pair],
+            # 'Hashstack V1': hashstack_v1_main_chart_data[current_pair],
+            NOSTRA_ALPHA: app.create_stablecoin_bundle(nostra_alpha)[current_pair],
+            NOSTRA_MAINNET: app.create_stablecoin_bundle(nostra_mainnet)[current_pair],
+        }
+        if current_pair == stable_coin_pair
+        else {
+            ZKLEND: zklend[current_pair],
+            # 'Hashstack V0': hashstack_v0_main_chart_data[current_pair],
+            # 'Hashstack V1': hashstack_v1_main_chart_data[current_pair],
+            NOSTRA_ALPHA: nostra_alpha[current_pair],
+            NOSTRA_MAINNET: nostra_mainnet[current_pair],
+        }
+    )
+    return protocol_main_chart_data_mapping
+
+
+def get_protocol_loans_data(zklend, nostra_alpha, nostra_mainnet):
+    protocol_loans_data_mapping = {
+        ZKLEND: zklend,
+        # 'Hashstack V0': hashstack_v0_loans_data,
+        # 'Hashstack V1': hashstack_v1_loans_data,
+        NOSTRA_ALPHA: nostra_alpha,
+        NOSTRA_MAINNET: nostra_mainnet,
+    }
+    return protocol_loans_data_mapping
+
+
+def noname_func(main_chart_data, current_pair, stable_coin_pair, collateral_token):
+    if current_pair == stable_coin_pair:
+        for stable_coin in src.settings.DEBT_TOKENS[:-1]:
+            debt_token = stable_coin
+            main_chart_data, collateral_token_price = app.process_liquidity(
+                main_chart_data, collateral_token, debt_token
+            )
+    else:
+        main_chart_data, collateral_token_price = app.process_liquidity(
+            main_chart_data, collateral_token, debt_token
+        )
+    return main_chart_data, collateral_token_price
+
+
+def get_usd_debt_boundaries(col1, streamlit, loans_data):
```

Comment:
This function is not returning anything, Maybe it make sense to rename it as `set_usd_debt_boundaries`